### DISCUSSION TITLE
date filter for current month, quarter, year, etc.

### DISCUSSION THREAD

**Discussion by jakejh** [apache/superset#28626] [Ideas]
A date filter has options for *previous* week, month, and year, but not *current* week, month, and year.

For example, if I want to filter for current year (i.e., year-to-date, 2024-01-01 to today), it seems like I have to use Advanced with custom start and end. It would be really convenient to have preset options for these.

**Comment by jgdoncel** [comment]
IMHO the option "current" for the date filter is a must. I know there are some workarounds but really complex for end users with low technical background.

**Comment by sfirke** [comment]
Closing as a duplicate of #21804 -- I clearly agree this is an important addition to make, just want to have discussion in a single place.  Feel free to add in details to what is discussed there.
